concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -370,44 +371,45 @@ def test_blender_verification_order_should_call_verification_result_with_result_
         )
 
     def test_that_blender_verification_order_should_work_properly_when_there_is_more_than_one_frame_to_render(self):
-        with mock.patch('verifier.tasks.download_archives_from_storage', autospec=True) as mock_download_archives_from_storage,\
```

Comment:
When doing something non obvious like this you should always remember to add a comment explaining why you're doing it.

Also, say which part is non-deterministic and fails.

Reply:
Should I comment this inside test?

Reply:
Updated.

Reply:
We should put more comments in code in general :)

Reply:
> Should I comment this inside test?

Yes.

> We should put more comments in code in general :)

Agreed. But please avoid comments that just say what you're doing. The comments should explain why and give context that's not available in the code they describe.